devices/Button/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
```

Comment:
Please check the other bindings, this fils should be much simpler. And also the values of the assembly should be correct. Not the default ones.

Reply:
Thanks, should be fixed!